NEWS.md

```diff
@@ -14,6 +14,7 @@
 
 ## Bug fixes
 
+- Setting `two_sd = TRUE` in `standardize()` and `standardize_parameters()` (correctly) on uses 2-SDs of the predictors (and not the response). 
 - `standardize_info()` / `standardize_parameters(method = "posthoc")` work for zero-inflated models ( #135 )
```

Comment:
@DominiqueMakowski FYI This wasn't working 100% correctly (especially with `method = "refit"`), probably for a long time - all fixed now. Just thought you'd like to know (: